tests/php/unit/Service/Appointments/MailServiceTest.php

```diff
@@ -505,6 +519,104 @@ public function testSendOrganizerBookingInformationEmail(): void {
 		$config = new AppointmentConfig();
 		$config->setUserId('test');
 		$config->setLocation('Test');
+		$config->setAvailabilityAsArray(
+			[
+				"timezoneId" => "Europe/Berlin",
+				"slots" => [
+				  "MO" => [["start" => 1713153660, "end" => 1713239940]],
+				  "TU" => [["start" => 1713153660, "end" => 1713239940]], 
+				  "WE" => [["start" => 1713153660, "end" => 1713239940]], 
+				  "TH" => [["start" => 1713153660, "end" => 1713239940]], 
+				  "FR" => [["start" => 1713153660, "end" => 1713239940]], 
+				  "SA" => [["start" => 1713153660, "end" => 1713239940]], 
+				  "SU" => [["start" => 1713153660, "end" => 1713239940]] 
+				] 
+			]
+		);
+		$this->userManager->expects(self::once())
+			->method('get')
+			->willReturn($this->createConfiguredMock(IUser::class, [
+				'getEmailAddress' => 'test@test.com',
+				'getDisplayName' => 'Test Test'
+			]));
+		$mailMessage = $this->createMock(IMessage::class);
+		$this->mailer->expects(self::once())
+			->method('createMessage')
+			->willReturn($mailMessage);
+		$mailMessage->expects(self::once())
+			->method('setFrom')
+			->willReturn($mailMessage);
+		$mailMessage->expects(self::once())
+			->method('setTo')
+			->willReturn($mailMessage);
+		$mailMessage->expects(self::once())
+			->method('useTemplate')
+			->willReturn($mailMessage);
+		$emailTemplate = $this->createMock(IEMailTemplate::class);
+		$this->mailer->expects(self::once())
+			->method('createEmailTemplate')
+			->willReturn($emailTemplate);
+		$emailTemplate->expects(self::once())
+			->method('addHeader');
+		$emailTemplate->expects(self::once())
+			->method('setSubject');
+		$emailTemplate->expects(self::once())
+			->method('addHeading');
+		$emailTemplate->expects(self::exactly(5))
+			->method('addBodyListItem');
+		$emailTemplate->expects(self::once())
+			->method('addFooter');
+		$this->mailer->expects(self::once())
+			->method('createEmailTemplate');
+		$this->mailer->expects(self::once())
+			->method('createAttachment');
+		$this->l10n->expects(self::exactly(6))
+			->method('t');
+		$this->lFactory->expects(self::once())
+			->method('findGenericLanguage')
+			->willReturn('en');
+		$this->lFactory->expects(self::once())
+			->method('get');
+		$this->dateFormatter->expects(self::once())
+			->method('formatDateTimeRelativeDay')
```

Comment:
by using `->with(args)` you can assert the value of the arguments passed to this method. It will allow you to test if the fourth argument matches the expected timezone value. Same for the new ``testSendOrganizerBookingInformationEmailDifferentTZ`` case